TFT/src/User/API/RRFStatusControl.c

```diff
@@ -30,6 +30,8 @@ void rrfStatusSet(char status)
           case 'I':
             // parseACK will take care of going to the print screen
             mustStoreCmd("M409 K\"job.file.fileName\"\n");
+            setPrintHost(true);
```

Comment:
This should probably be deferred until the response of M409 before we start the print status display?

I would think along the lines of a new function in `gcode.c` like `rrf_request_job_name(...)` that would invoke its own parser and launch the job